tasks/release.py

```diff
@@ -1815,3 +1816,17 @@ def create_schedule(_, version, freeze_date):
         raise Exit(f"You must set {required_environment_variables} environment variables to use this task.", code=1)
     release_page = create_release_page(version, date.fromisoformat(freeze_date))
     print(f"Release schedule pages {release_page['url']} {color_message('successfully created', 'green')}")
+
+
+@task
+def chase_release_managers(_, version):
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
I like the name